### PR TITLE
[Issue 508] Renamed IT tests

### PR DESCRIPTION
**Pull request by echernous**
Signed-off-by: Kate Chernousova <eachernous@gmail.com>

The change addresses https://github.com/strimzi/strimzi-kafka-bridge/issues/508, renamed some of tests as ITs.

### CODE REVIEW

**Review by scholzj — COMMENTED** [review]
Are the IT tests being executed? I think you migth need to add the failsafe plugin to the `pom.xml` to run them.

**Comment by Frawless** [comment]
Yes, it looks that tests weren't executed. You can reuse the way how we handle ITs in operator repo with failsafe plugin.

**Comment by echernous** [comment]
Hi @scholzj and @Frawless, thanks for checking! Added the failsafe plugin now

File: pom.xml [hunk 1]
```diff
@@ -272,25 +276,53 @@
...
+                                        <execution>
+                                                <goals>
+                                                        <goal>integration-test</goal>
+                                                        <goal>verify</goal>
+                                                </goals>
+                                                <configuration>
+                                                        <includes>
+                                                                <include>**/IT*.java</include>
+                                                                <include>**/*IT.java</include>
+                                                                <include>**/*StaticTest.java</include>
```
Comment (Frawless) [inline pom.xml:297]: Why this one is not named as the others ITs?

**Review by Frawless — APPROVED** [review]
LGTM, I can see ITs executed in travis build for jdk8 and jdk11.

File: pom.xml (hunk 1 above)
Comment (echernous) [reply pom.xml:297]: It should be `StaticTests` but I missed "s" in the end. I assume this class was created to put some static tests but currently there is only one test, should it be named as BridgeStartupIT or just StaticIT?

Comment (echernous) [reply pom.xml]: renamed it to StaticIT to be consistent

File: pom.xml [hunk 2]
```diff
@@ -19,6 +19,7 @@
 		<hamcrest.version>2.2</hamcrest.version>
 		<maven-jar-plugin.version>3.1.2</maven-jar-plugin.version>
 		<maven-surefire-plugin.version>3.0.0-M5</maven-surefire-plugin.version>
+                <maven-failsafe-plugin.version>3.0.0-M5</maven-failsafe-plugin.version>
```
Comment (scholzj) [inline pom.xml:22]: Should this have the same alignment as the other lines? Or is it misaligned for some reason only in the diff?

File: pom.xml [hunk 3]
```diff
@@ -30,6 +31,9 @@
 		<jmx-prometheus-collector.version>0.12.0</jmx-prometheus-collector.version>
 		<commons-cli.version>1.4</commons-cli.version>
 		<test-container.version>0.20.0</test-container.version>
+                <!-- property to skip surefire tests during failsafe execution -->
+                <!--suppress UnresolvedMavenProperty -->
+                <skip.surefire.tests>${skipTests}</skip.surefire.tests>
```
Comment (scholzj) [inline pom.xml:36]: Same as above.

**Review by scholzj — COMMENTED** [review]
The code looks good to me. But the pom.xml seems to have a weird alignment. Not sure it is just the GitHub diff or also in reality. Could you please check it?

**Review by sknot-rh — APPROVED** [review]

File: pom.xml (hunk 2 above)
Comment (echernous) [reply pom.xml]: yeah, something went wrong there, fixed now, thanks!

**Review by scholzj — APPROVED** [review]

**Pull request merged**